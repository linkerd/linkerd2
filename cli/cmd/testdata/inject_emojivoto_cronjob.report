
cronjob "hello" annotated
cronjob "hello" injected

