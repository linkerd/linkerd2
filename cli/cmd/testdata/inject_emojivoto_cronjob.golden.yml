apiVersion: batch/v1beta1
kind: CronJob
metadata:
  name: hello
  namespace: emojivoto
spec:
  jobTemplate:
    spec:
      template:
        metadata:
          annotations:
            linkerd.io/inject: enabled
          labels:
            foo: bar
        spec:
          containers:
          - args:
            - /bin/sh
            - -c
            - date; echo Hello from the Kubernetes cluster
            image: busybox
            name: hello
          restartPolicy: OnFailure
  schedule: '*/10 * * * *'
---
